### PR TITLE
mycli.md: add page

### PR DESCRIPTION
**Pull request by jsonbruce**
An alternative tool for mysql.

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/mycli.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# mycli
+
+> A CLI for MySQL, MariaDB, and Percona with auto-completion and syntax highlighting.
+
+- Connect to a database with current login user:
```
Comment (sbrl) [inline pages/linux/mycli.md]: Perhaps ...`with the currently logged in user` would read better?

File: pages/linux/mycli.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`mycli {{database_name}}`
+
+- Connect to a database with specified user:
```
Comment (sbrl) [inline pages/linux/mycli.md]: Perhaps ...`with the specified`... would read better?

File: pages/linux/mycli.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`mycli -u {{user}} {{database_name}}`
+
+- Connect to a database with specified user and host:
```
Comment (sbrl) [inline pages/linux/mycli.md]: Perhaps `Connect to a database on the specified host with the specified user` would read better?

File: pages/linux/mycli.md (hunk 2 above)
Comment (jsonbruce) [reply pages/linux/mycli.md]: It's right.

File: pages/linux/mycli.md (hunk 3 above)
Comment (jsonbruce) [reply pages/linux/mycli.md]: How do you think
 `Connect to a database with the specified user and host` ?

File: pages/linux/mycli.md (hunk 2 above)
Comment (sbrl) [reply pages/linux/mycli.md]: Erm it's right? Right how?

File: pages/linux/mycli.md (hunk 3 above)
Comment (sbrl) [reply pages/linux/mycli.md]: I'm not sure I understand the question.

Comment (jsonbruce) [reply pages/linux/mycli.md]: I think just one `specified` in the sentence is OK.
I'm not sure that `Connect to a database with the specified user and host` will be missunderstanding.

**Comment by jsonbruce** [comment]
Anyway, I've completed the modification. PTAL.

File: pages/linux/mycli.md (hunk 3 above)
Comment (sbrl) [reply pages/linux/mycli.md]: Ahhh. Sure :-)

**Comment by sbrl** [comment]
Looks ok to me! Thanks again, @jsonbruce :D

**Pull request merged**